### PR TITLE
#CU-81tm1p add override color to box

### PR DESCRIPTION
**Pull request by binhtt13122000**
* I will fix issue in the slack, if the bgcolor is undefined, the background color is same with its parent.
* To using backgroundColor default (white), I add new type to bgcolor is default.
* I add new props is overrideColor,  because I want to my backgroundColor is A so the color is onA, but Kien's task and some task need to using background color is "primary" but color is "secondary" so I create overrideColor to do that. But I do not recommend using it because it's unsafe. It have lots of case (normal color, theme color, text color,...) so that I can't make it be more safe.
image

### CODE REVIEW

**Comment by binh1298** [comment]
Task linked: [CU-81tm1p Box Component](https://app.clickup.com/t/81tm1p)

**Comment by binh1298** [comment]
Why not just use color: "transparent"?

**Comment by binhtt13122000** [comment]
transparent is set when bgcolor is undefined. And default is alias of background.default of reply theme.
image

**Comment by binh1298** [comment]
Is this ready for review?

File: src/components/Boxs/BoxBase/index.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export interface BoxBaseProps extends MaterialBoxProps {
         | "initial"
         | "inherit"
         | "unset";
-    bgcolor?: "primary" | "secondary" | "info" | "error" | "warning" | "success";
+    bgcolor?: "primary" | "secondary" | "info" | "error" | "warning" | "success" | "common";
```
Comment (binh1298) [inline src/components/Boxs/BoxBase/index.tsx:79]:
```suggestion
    bgcolor?: "primary" | "secondary" | "info" | "error" | "warning" | "success" | undefined;
```

**Review by binh1298 — APPROVED** [review]

**Pull request merged**